_includes/cloudcode/cloud-code.md

```diff
@@ -672,6 +672,9 @@ Parse.Cloud.afterFind(Parse.User, async (request) => {
 })
 ```
 
+### Some considerations to be aware of
```

Comment:
If we want to make developers aware of security implications, I think we should make a broader change in the docs.

1) Move this to the already existing `Using the Master Key in cloud code` section
2) Refactor the existing section to make it more prominent
3) Articulate your example in a more general way


So instead of this:
----
# Using the Master Key in cloud code
Set `useMasterKey:true` in the requests that require master key.

## Examples:

```javascript
query.find({useMasterKey:true});
object.save(null,{useMasterKey:true});
Parse.Object.saveAll(objects,{useMasterKey:true});
```
----

I suggest we rename this verbose headline to `Security`.

----
# Security
## Master Key
To override object and class access permissions, you can set `useMasterKey: true` if the request accepts the master key option.

### Examples

```javascript
query.find({ useMasterKey: true });
```

```javascript
object.save(null, { useMasterKey: true });
```

```javascript
Parse.Object.saveAll(objects, { useMasterKey: true });
```

### Considerations
- If you set `masterKey: true` when fetching objects with a query or relation in [Cloud Functions](https://docs.parseplatform.org/cloudcode/guide/#cloud-functions) or [Find Triggers](https://docs.parseplatform.org/cloudcode/guide/#find-triggers), the complete object will be returned. You may want to remove object properties that the client should not be able to access before sending it on to the client.
----